app/src/main/java/com/babylon/wallet/android/domain/usecases/GetDAppWithResourcesUseCase.kt

```diff
@@ -14,7 +16,8 @@ class GetDAppWithResourcesUseCase @Inject constructor(
 
     suspend operator fun invoke(
         definitionAddress: AccountAddress,
-        needMostRecentData: Boolean
+        needMostRecentData: Boolean,
+        verifyResources: Boolean = true
```

Comment:
I am not sure we need this one. I think we always need to verify the resources. If we ever need the rest of them we can add this, but for now I believe it is unnecessary.

Reply:
yes, there are only 2 references of this use case usage and appear that in those call sites we want to do a validation. Will update this